test/qa/lib/login.py

```diff
@@ -88,51 +88,28 @@ def browser_element_interact(self, xpath: str, write:str=None, return_attribute:
             return None
 
 
-def get_default_credentials():
-    """Returns tuple[username,token]."""
-    default_username = os.environ.get("DEFAULT_LOGIN_USERNAME")
-    default_token = os.environ.get("DEFAULT_LOGIN_TOKEN")
-    ci_credentials = os.environ.get("NA_TESTS_CREDENTIALS")
-    if ci_credentials is not None:
-        devs = json.loads(ci_credentials)
-        dev_email = os.environ.get("GITLAB_USER_EMAIL")
-        if dev_email in devs:
-            default_username = devs[dev_email]["username"]
-            default_token = devs[dev_email]["token"]
-    return default_username, default_token
+def get_token(key):
+    """Returns token by a given key."""
+    na_credentials = os.environ.get("NA_TESTS_CREDENTIALS")
+    na_credentials_key = os.environ.get("NA_CREDENTIALS_KEY")
```

Comment:
when will `NA_CREDENTIALS_KEY` be used?

Reply:
If you want to use different credentials for some reason. e. g. `default_test_some_category`. In our case it would be meshnet tests